.github/workflows/ets-from-source.yml

```diff
@@ -0,0 +1,83 @@
+name: Test with ETS packages from source
+
+on:
+  pull_request:
```

Comment:
This will be removed before the PR is merged IIUC. Right?

Reply:
yep 👍